webapp/components/login/login_controller.jsx

```diff
@@ -51,6 +51,10 @@ export default class LoginController extends React.Component {
             showMfa: false,
             loading: false
         };
+
+        if (this.props.location.query.extra === Constants.SIGNIN_VERIFIED && this.props.location.query.email) {
+            this.state.loginId = this.props.location.query.email; // eslint-disable-line react/no-direct-mutation-state
```

Comment:
Instead of disabling ESLint here, you could change the above to:
```
let loginId = '';
if (this.props.location.query.extra === Constants.SIGNIN_VERIFIED && this.props.location.query.email) {
    loginId = this.props.location.query.email;
}

this.state = {
    ...
    loginId,
    ...
};
```

We'd rather not encourage disabling ESLint rules if we can avoid it.